### PR TITLE
build(deps): bump versions

### PR DESCRIPTION
**Pull request by driskull**
build(deps): bump versions

### CODE REVIEW

**Comment by benelan** [comment]
should we remove `typescript` here too:
https://github.com/Esri/calcite-components/blob/master/.github/dependabot.yml#L18

**Comment by driskull** [comment]
> should we remove typescript here too:

Yeah, lets see if this works. Last time I tried bumping it storybook was having issues.

**Comment by driskull** [comment]
@benelan @jcfranco anyone able to debug webpack storybook issues? The PR seems to pass build but I can't get storybook to run locally. I see this error:

```
 ｢atl｣: Using typescript@4.4.4 from typescript
ℹ ｢atl｣: Using tsconfig.json from /Users/dris0000/Sites/git/calcite-components/tsconfig-storybook.json
✖ ｢atl｣: Child process failed to process the request: Error: Debug Failure. False expression.
    at resolveNamesWithLocalCache (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:116384:26)
    at Object.resolveModuleNames (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:116427:20)
    at Object.resolveModuleNames (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:118165:112)
    at actualResolveModuleNamesWorker (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:111711:133)
    at resolveModuleNamesWorker (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:111977:26)
    at resolveModuleNamesReusingOldState (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:112074:24)
    at processImportedModules (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:113532:35)
    at findSourceFileWorker (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:113333:17)
    at findSourceFile (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:113194:26)
    at processImportedModules (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:113568:25)
    at findSourceFileWorker (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:113333:17)
    at findSourceFile (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:113194:26)
    at processImportedModules (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:113568:25)
    at findSourceFileWorker (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:113333:17)
    at findSourceFile (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:113194:26)
    at processImportedModules (/Users/dris0000/Sites/git/calcite-components/node_modules/typescript/lib/typescript.js:113568:25)
✖ ｢atl｣: Child process failed to process the request: TypeError: Cannot read properties of undefined (reading 'updateRootFileNames')
    at getProgram (/Users/dris0000/Sites/git/calcite-components/node_modules/awesome-typescript-loader/src/checker/runtime.ts:152:10)
    at getEmitOutput (/Users/dris0000/Sites/git/calcite-components/node_modules/awesome-typescript-loader/src/checker/runtime.ts:403:19)
    at emit (/Users/dris0000/Sites/git/calcite-components/node_modules/awesome-typescript-loader/src/checker/runtime.ts:422:24)
    at processEmit (/Users/dris0000/Sites/git/calcite-components/node_modules/awesome-typescript-loader/src/checker/runtime.ts:458:22)
    at /Users/dris0000/Sites/git/calcite-components/node_modules/awesome-typescript-loader/src/checker/runtime.ts:610:6
    at Object.send (/Users/dris0000/Sites/git/calcite-components/node_modules/awesome-typescript-loader/src/checker/runtime.ts:49:6)
    at Checker.req (/Users/dris0000/Sites/git/calcite-components/node_modules/awesome-typescript-loader/src/checker/checker.ts:100:15)
    at Checker.emitFile (/Users/dris0000/Sites/git/calcite-components/node_modules/awesome-typescript-loader/src/checker/checker.ts:105:15)
    at transform (/Users/dris0000/Sites/git/calcite-components/node_modules/awesome-typescript-loader/src/index.ts:130:26)
    at transformationFunction (/Users/dris0000/Sites/git/calcite-components/node_modules/awesome-typescript-loader/src/index.ts:72:40)
    at compiler (/Users/dris0000/Sites/git/calcite-components/node_modules/awesome-typescript-loader/src/index.ts:83:21)
    at Object.loader (/Users/dris0000/Sites/git/calcite-components/node_modules/awesome-typescript-loader/src/index.ts:14:12)
    at LOADER_EXECUTION (/Users/dris0000/Sites/git/calcite-components/node_modules/loader-runner/lib/LoaderRunner.js:119:14)
    at runSyncOrAsync (/Users/dris0000/Sites/git/calcite-components/node_modules/loader-runner/lib/LoaderRunner.js:120:4)
    at iterateNormalLoaders (/Users/dris0000/Sites/git/calcite-components/node_modules/loader-runner/lib/LoaderRunner.js:232:2)
    at Array.<anonymous> (/Users/dris0000/Sites/git/calcite-components/node_modules/loader-runner/lib/LoaderRunner.js:205:4)
[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "null".] {
  code: 'ERR_UNHANDLED_REJECTION'
}
```

**Comment by benelan** [comment]
Storybook also doesn't run via the gh action:
https://github.com/Esri/calcite-components/runs/4195575760?check_suite_focus=true#step:5:85

File: .storybook/utils.ts [hunk 1]
```diff
@@ -121,17 +115,7 @@ export const createComponentHTML = (
     })
     .join(" ")}>${contentHTML}</${tagName}>`;
 
-export const globalDocsPage: typeof DocsPage = () => (
-  <React.Fragment>
```
Comment (driskull) [inline .storybook/utils.ts:125]: We can also hide the title using CSS. 

```css
.sbdocs-h1 { display:none; }
```

**Pull request closed without merging**